inst/pkgdown/_pkgdown.yml

```diff
@@ -0,0 +1,47 @@
+template:
+  includes:
+    before_title: '<a href="https://cynkra.com"><img src="https://cynkra.com/assets/images/favicon-16x16.png" id="cynkra-icon" alt="cynkra"></a>'
+  bootstrap: 5
+  bslib:
+    bg: "white"
+    fg: "black"
+    primary: "#6366f1"
+    link-color: "#184FA9"
+    base_font: "frutiger-light,sans-serif"
+    code_font: {google: "Fira Mono"}
+
+footer:
+  structure:
+    left: developed_by
+    right: cynkra
+  components:
+    cynkra: <p>Sponsored by <a class="cynkra-logo-font-footer" href="https://www.cynkra.com">cynkra</a><a class="cynkra-logo-footer" href="https://www.cynkra.com">&#9632;</a>. Visit us at <a href="https://www.cynkra.com/">www.cynkra.com</a>.</p>
```

Comment:
it seems easier than having a template HTML, so IMHO it's ok not to have the package name here. Thoughts?